development/tools/cmd/pjtester/README.md

```diff
@@ -80,7 +86,9 @@ This is the Prow Job tester flow:
 
 ### Execution of any code without review?
 
-This was the main requirement for this tool. However, we did put some security in place. The `pre-main-test-infra-pjtester` Prow Job is running on the `trusted-workload` cluster, where it has everything it needs for succesful execution. Every Prow Job to test will be scheduled on the `untrusted-workload` cluster, where no sensitive data exists. As for any other PR from a non-Kyma-organization member, every test must be triggered manually.
+This was the main requirement for this tool. However, we did put some security in place. The `pre-main-test-infra-pjtester` Prow Job is running on the `trusted-workload` cluster, where it has everything it needs for successful execution. Every Prow Job to test will be always scheduled on the `untrusted-workload` cluster, where no sensitive data exists. As for any other PR from a non-Kyma-organization member, every test must be triggered manually.
+
+To prevent overriding existing GitHub contexts results on open pull requests with results of execution of test prowjobs, pjtester add its prefix to context defined in prowjob definition. This way test prowjobs have always it's own context name.
```

Comment:
```suggestion
To prevent overriding existing GitHub contexts results on open pull requests with results of execution of test prowjobs, pjtester adds its prefix to the context defined in the prowjob definition. This way the test prowjobs have always its own context name.
```